haskell-ide-engine.cabal

```diff
@@ -62,12 +62,13 @@ library
                      , directory
                      , either
                      , fast-logger
-                     , ghc >= 7.10.2 && < 7.11
-                     , ghc-mod >= 5.4
+                     , ghc >= 7.10.2 && < 8.3
```

Comment:
If we add upperbounds, shouldn’t this be < 8.2 or <= 8.0? It seems unlikely that 8.2 will not break anything.